official/utils/testing/reference_data.py

```diff
@@ -261,12 +261,19 @@ def _evaluate_test_case(self, name, graph, ops_to_eval, correctness_function):
             .format(differences, tf_version_comparison))
 
       eval_results = [op.eval() for op in ops_to_eval]
+      print("eval_resuls:{}".format(eval_results))
```

Comment:
do you want to leave this prints here or were they only for debugging?

Reply:
Removed